server/channels/app/platform/license.go

```diff
@@ -49,6 +49,8 @@ func (ps *PlatformService) License() *model.License {
 }
 
 func (ps *PlatformService) LoadLicense() {
+	c := request.EmptyContext(ps.logger)
```

Comment:
I'm assuming this is safe to use around outside tests?

Reply:
Yes, it is. There are plenty of places where it's used.